src/index.tmpl

```diff
@@ -1,10 +1,19 @@
 <!DOCTYPE html>
 <html>
-  <head>
-    <meta charset="UTF-8">
-    <title><%=htmlWebpackPlugin.options.title %></title>
-  </head>
-  <body>
-    <div id="app">React Application placeholder</div>
-  </body>
-</html>
+	<head>
+		<meta charset="UTF-8">
+		<title><%=htmlWebpackPlugin.options.title %></title>
+		<link rel="stylesheet" type="text/css" href="../node_modules/bootstrap/dist/css/bootstrap.css">
```

Comment:
I guess that bootstrap should be included somehow different but let us keep it as is now. I will investigate it and fix if needed